Language/Variables/Implicitly_Induced_Getters_and_Setters/definition_A06_t02.dart

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Case ⟨Setter: Late-final variable with declared type ⟩. A
+/// variable declaration of the form static? late final T id; implicitly induces
+/// a setter with the header void set id(T x). If this setter is executed in a
+/// situation where the variable id has not been bound, it will bind id to the
+/// object that x is bound to. If this setter is executed in a situation where
+/// the variable id has been bound to an object, a dynamic error occurs
+///
+/// @description Checks that it is a dynamic error if this setter is executed in
+/// a situation where the variable has been bound to an object
```

Comment:
Perhaps?:
```suggestion
/// @description Checks that it is a dynamic error if this getter is executed in
/// a situation where the variable has not been bound to an object
```

Reply:
Ups, my fault. Thank you! Test updated to fit the description (and the assertion)